docs/NooBaaNonContainerized/Health.md

```diff
@@ -269,6 +274,15 @@ Output:
         }
       ]
     },
+    "notif_storage_threshold_details": {
+      "invalid_storages": [],
```

Comment:
do we need `invalid_storages`? I am not sure invalid and valid are applicable here, Since we will have only one path for the log directory. Usually follow this structure when there are possible invalid and valid scenarios, But here I dont think thats the case.